modules/scripts-utils/dtbook-utils/src/main/resources/xml/dtbook-utils-library.xpl

```diff
@@ -20,5 +20,6 @@
     <p:import href="dtbook-update-links.xpl"/>
 
     <p:import href="fix-dtbook/fix-dtbook.xpl"/>
+    <p:import href="fix-dtbook/doctyping.xpl"/>
```

Comment:
We already had functionality in Pipeline to set the doctype (with `px:fileset-store`). We're including the step it in this release but we're going to clean it up afterwards.